app/assets/javascripts/components/story/ExpandedStory/ExpandedStoryDescription.js

```diff
@@ -13,12 +14,17 @@ class ExpandedStoryDescription extends React.Component {
     };
 
     this.toggleField = this.toggleField.bind(this);
+    this.mentionFormat = this.mentionFormat.bind(this);
   };
 
   toggleField() {
     this.setState({ editing: !this.state.editing });
   };
 
+  mentionFormat(id, display) {
```

Comment:
The content of this variable is not the format of a mention, but a function that formats a mention, thus we should call it `formatMention` to make it clearer, what do you think?